services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The import statement for `User` is unnecessary because you're using `get_user_model()` to interact with the user model. Consider removing this import to clean up the code.